projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/r2/liferay-r2/css-parse.js

```diff
@@ -584,7 +584,13 @@ module.exports = function (css, options) {
 
 	function _atrule(name) {
 		var pos = position();
-		var m = match(new RegExp('^@' + name + ' *([^;\\n]+);'));
+		var m = match(
+			new RegExp(
+				'^@' +
+					name +
+					' *(?:url\\(([^)]+)\\)|([^;\\n]*))(?: *([^;\\n]*))?;'
```

Comment:
I'm terrible at reviewing regex, but my semi-trustworthy friend seems to indicate that this should be fine
<img width="779" alt="Screenshot 2024-03-07 at 10 34 50 AM" src="https://github.com/liferay/liferay-frontend-projects/assets/6843530/8edff075-58ea-460d-be8c-ff30d45fe01f">